docs/dynamo.md

```diff
@@ -86,5 +86,15 @@ average | 1.0779
 
 We are currently working on the optimizer support and that will be availiable on nightly soon but won't be in the 2.0 release.
 
+### Feature gaps
+There are few gaps we want to call out that are preventing us from using the TorchDynamo on larger scale models.
+
+1. TorchDynamo does not trace the communication ops(like `all_reduce` and `all_gather`) yet. It will let the communication ops run on eager mode, which is not ideal for PyTorch/XLA use case since we would want the compiler to see all communication ops in the same graph.
+2. TorchDynamo will trace forward and backward into separate graphs. For PyTorch/XLA it is important to let the XLA compiler see the whole step as one graph to best optimize the speed. There is also a fixed overhead to launch every device execution which make executing multiple graphs per training step less ideal.
+
+These feature gaps compared to Lazy Tensor makes it less efficient in real world training use cases, especially the tracing cost can be overlapped with the execution in training.
+
 ### Take away
-TorchDynamo provides a really promising way for the compiler backend to hide the complexity from the user and easily retrieve the modeling code in a graph format. Compared with PyTorch/XLA’s traditional Lazy Tensor way of extracting the graph, TorchDynamo can skip the graph tracing for every iteration hence provide a much better inference response time. However TorchDynamo does not trace the communication ops(like `all_reduce` and `all_gather`) yet and it provides separate graphs for the forward and the backward which hurts xla performance. These feature gaps compared to Lazy Tensor makes it less efficient in real world training use cases, especially the tracing cost can be overlapped with the execution in training. The PyTorch/XLA team will keep investing in TorchDynamo and work with upstream to mature the training story.
\ No newline at end of file
+TorchDynamo provides a really promising way for the compiler backend to hide the complexity from the user and easily retrieve the modeling code in a graph format. Compared with PyTorch/XLA's traditional Lazy Tensor way of extracting the graph, TorchDynamo can skip the graph tracing for every iteration, hence providing a much better inference response time.
```

Comment:
TorchDynamo provides a really promising way for the compiler backend to hide the complexity from the user and easily retrieve the modeling code in a graph format. Compared with PyTorch/XLA's traditional LazyTensorCore method of graph extraction, TorchDynamo can skip the graph tracing for every iteration, hence providing a much better inference response time.